bither-ios/es.lproj/Localizable.strings

```diff
@@ -0,0 +1,735 @@
+
+/* No comment provided by engineer. */
+"Private keys in danger." = "LLaves privadas en peligro.";
+
+/* No comment provided by engineer. */
+"QR Code" = "Código QR";
+
+/* No comment provided by engineer. */
+"QR Code saved." = "Código QR grabado.";
+
+/* No comment provided by engineer. */
+"Received" = "Recibido";
+
+/* No comment provided by engineer. */
+"Received:" = "Recibido:";
+
+/* No comment provided by engineer. */
+"Scan %@" = "Escanear %@";
+
+/* No comment provided by engineer. */
+"Scan %@\nPage %d, Total %d" = "Escanear %1$@\nPágina %2$d, Total %3$d";
+
+/* No comment provided by engineer. */
+"Scan Bitcoin Address" = "Escanear dirección de Bitcoin";
+
+/* No comment provided by engineer. */
+"Scan Bither Cold to sign" = "Escanear Bither Frío para firmar";
+
+/* No comment provided by engineer. */
+"Scan by clone destination" = "Escanear por por destino clonado";
+
+/* No comment provided by engineer. */
+"Scan Private Key QR Code" = "Escanear código QR privado";
+
+/* No comment provided by engineer. */
+"Scan QR Code for Bitcoin address" = "Escanear código QR para dirección de Bitcoin";
+
+/* No comment provided by engineer. */
+"Scan QR code No.1 provided by Bither" = "Escanear código QR No.1 proveido por Bither";
+
+/* No comment provided by engineer. */
+"Scan The Clone Source" = "Escanear la fuente clon";
+
+/* No comment provided by engineer. */
+"Scan to watch Bither Cold" = "Escanear para observar Bither Frío";
+
+/* No comment provided by engineer. */
+"Scan unsigned transaction failed" = "Falló el escaneo de transacción sin firmar";
+
+/* No comment provided by engineer. */
+"Scan Unsigned TX" = "Escaneo de TX sin firmar";
+
+/* No comment provided by engineer. */
+"Scan with Bither Cold" = "Escanear con Bither Frío";
+
+/* No comment provided by engineer. */
+"Scan with Bither Hot to sign tx" = "Escanear con Bither Caliente para firmar TX";
+
+/* No comment provided by engineer. */
+"Scan with Bither Hot to watch Bither Cold" = "Escanear con Bither Caliente para vigilar Bither Frío";
+
+/* No comment provided by engineer. */
+"Select an address to donate" = "Seleccione una dirección con la que desee donar";
+
+/* No comment provided by engineer. */
+"Sell: %@" = "Vender: %@";
+
+/* No comment provided by engineer. */
+"Send" = "Enviar";
+
+/* No comment provided by engineer. */
+"Send failed." = "Envio falló.";
+
+/* No comment provided by engineer. */
+"Send success." = "Envio exitoso.";
+
+/* No comment provided by engineer. */
+"Send:" = "Enviar:";
+
+/* No comment provided by engineer. */
+"Sent" = "Enviado";
+
+/* No comment provided by engineer. */
+"Set Avatar" = "Fijar Avatar";
+
+/* No comment provided by engineer. */
+"Set Password" = "Fijar contraseña";
+
+/* No comment provided by engineer. */
+"Sign Transaction" = "Firmar Transacción";
+
+/* No comment provided by engineer. */
+/* suspicious error repeated? signing transaction*/
+"Sign Trasaction" = "Firmando transacción";
```

Comment:
Thank you for pointing out this misspelling. This is our fault. You may delete this line now. @zzaappedd

Reply:
suspicious error repeated? signing transaction